house/migrations/0003_delete_job_alter_city_id_alter_country_id_and_more.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.0.3 on 2022-03-27 11:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('house', '0002_add_initial_data'),
```

Comment:
what is the second dependency? 0002_add_initial_data
On which PR is it?
#26 was merged. Does the PR still depend on another PR?

Reply:
It was created because of the Foreign key relations with house